### DISCUSSION TITLE
22.11

### DISCUSSION THREAD

**Discussion by falkTX** [DISTRHO/Cardinal#438] [Announcements]
(release notes to be written, please come back later!)

<hr /><em>This discussion was created from the release <a href='https://github.com/DISTRHO/Cardinal/releases/tag/22.11'>22.11</a>.</em>

**Comment by N1c01a1** [comment]
Hi, Studio One 5 crashes when loading up cardinal, does install fine

**Reply by subderground** [reply]
Hi, Personally, (after deleted the older one) I always just copy/paste the chosen folder (VST/LV2/VST3 version) in the right destination folder by hand/mouse. It works perfectly. Perhaps, you could try without the installer ? 

Ps:  The last nightly works great too :-) Thanks a lot for all that new modules !!!

**Comment by mervenca** [comment]
It says on the main page that Surge XT is included, but I downloaded the 22.11 version and don't see them.. Am I doing something wrong or is it actually missing in the release?

**Reply by dromer** [reply]
It's included in the _current_ builds. Which is set to release later this month.

You can help test the upcoming release by downloading builds from the Actions tab -> https://github.com/DISTRHO/Cardinal/actions

**Reply by mervenca** [reply]
Oh, okay, thanks! I guess the term "current" threw me off a little bit, not particularly familiar with github..
Would love to test it out, but cant find any downloadable content in your link? Or am I supposed to compile it somehow myself?
Sorry if I'm asking too noob questions, can also wait until in releases :)
But love your work, Cardinal is excellent!

**Reply by falkTX** [reply]
this is explained in https://github.com/DISTRHO/Cardinal/wiki/Install

> You can find builds for pretty much any recent Cardinal commit [here](https://github.com/DISTRHO/Cardinal/actions/workflows/build.yml).
> Just click on any successful build, and scroll to the bottom to find the builds.
> (note the canvas-like area in the middle prevents mouse wheel scrolling)

**Comment by tvrnxr** [comment]
Hi,

Any attempt to use VST3 Cardinal Plugins in Ableton 10.1.14 results in my entire program crashing. VST2 is working fine though! I'm not much of a developer / troubleshooter but I've attached an Ableton crash report in case it's of any help:


[Ableton Crash Report 2022-12-19 130634 Live 10.1.14.zip](https://github.com/DISTRHO/Cardinal/files/10259507/Ableton.Crash.Report.2022-12-19.130634.Live.10.1.14.zip)
